.github/workflows/ci.yml

```diff
@@ -642,6 +642,43 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  unit-benchmark:
+    name: Unit Benchmarks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+
+      - name: Setup Cache for Rust and Benchmarks
+        uses: Swatinem/rust-cache@v2
+        with:
+          # Additional non workspace directories, separated by newlines
+          cache-directories: ./benches
```

Comment:
CI is complainig about: 
```
Unexpected input(s) 'cache-directories', valid inputs are ['shared-key', 'key', 'env-vars', 'workspaces', 'cache-on-failure']
```

Reply:
Yeah this warning took me by surprise here: https://github.com/infinyon/fluvio/actions/runs/3373039401/jobs/5597164655#step:4:1

Because I can see in the docs and also in their input implementation that `cache-directories` is actually supported 🤔 .

https://github.com/Swatinem/rust-cache#example-usage